.classpath

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Adding generic comment for Complete PR

1.Make changes for page object model
2.Variable names should be camel cased
3.No class path and .class should be part of PR
4.No object initialization at test case level
5.Remove all comment before raising PR
6.Variable Names should should be meaning-full